src/Bkwld/Croppa/Helpers.php

```diff
@@ -68,17 +68,33 @@ public function tag($url, $width = null, $height = null, $options = null) {
 	}
 
 	/**
-	 * Pass through URL requrests to URL->generate().
+	 * Pass through URL requests to URL->generate().
 	 *
 	 * @param string $url URL of an image that should be cropped
 	 * @param integer $width Target width
 	 * @param integer $height Target height
-	 * @param array $options Addtional Croppa options, passed as key/value pairs.  Like array('resize')
+	 * @param array $options Additional Croppa options, passed as key/value pairs.  Like array('resize')
 	 * @return string The new path to your thumbnail
 	 * @see Bkwld\Croppa\URL::generate()
 	 */
 	public function url($url, $width = null, $height = null, $options = null) {
 		return $this->url->generate($url, $width, $height, $options);
 	}
 
+
+	/**
+	 * Render image
+	 *
+	 * @param string $url URL of an image that should be cropped
+	 * @param integer $width Target width
+	 * @param integer $height Target height
+	 * @param array $options Additional Croppa options, passed as key/value pairs.  Like array('resize')
+	 * @return string The new path to your thumbnail
+	 * @see Bkwld\Croppa\URL::generate()
+	 */
+	public function render($croppaurl) {
+		global $app;
+		return $app['Bkwld\Croppa\Handler']->render($croppaurl);
```

Comment:
I've been trying to use dependency injection for all classes and referring to the app binding here breaks that.  Instead, I think `Handler` should be injected into the constructor and used here.  Sucks to have another dependency, but /shrug